packages/swapper/src/swappers/thorchain/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -60,19 +58,22 @@ export const getThorTradeQuote: GetThorTradeQuote = async ({ deps, input }) => {
         details: { buyAssetId }
       })
 
-    const priceRatio = await getPriceRatio(deps, { sellAssetId, buyAssetId })
-    const rate = bnOrZero(1).div(priceRatio).toString()
-    const buyAmount = normalizeAmount(bnOrZero(sellAmount).times(rate))
+    const tradeRate = await getTradeRate(sellAsset, buyAsset.assetId, sellAmount, deps)
```

Comment:
getTradeRate gets the exact rate given an amount. more accurate than using getPriceRatio. changed this in a couple places